### PR TITLE
don't charge 0.99 for zippy (bug 1123350)

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

File: lib/solitude/tests.py [hunk 1]
```diff
@@ -635,6 +640,12 @@ def test_callback_validation_failure(self):
         is_valid = self.provider.is_callback_token_valid({'foo': 'bar'})
         eq_(is_valid, False)
 
+    @raises(ValueError)
+    def test_no_usd(self):
+        self.provider.provider.create_transaction(
+            None, None, None, None, None, None, None,
+            {}, None, None, None, None)
```
Comment (kumar303) [inline lib/solitude/tests.py]: can't you just use ``self.configure(prices=None)`` or something along those lines?

**Comment by kumar303** [comment]
this commit message is misleading but I get it.

r+wc

File: lib/solitude/tests.py (hunk 1 above)
Comment (andymckay) [inline lib/solitude/tests.py]: I can! Thanks, much more sane.

**Comment by muffinresearch** [comment]
giphy

**Pull request merged**